utils/errorHandler.js

```diff
@@ -0,0 +1,14 @@
+const errorHandler = (err, req, res, next) => {
```

Comment:
This handler in the express is middleware, so it would be more correct for an architecture of your project to move it to the middleware folder.